### PR TITLE
Capitalised CSS (subcategory) & FreeCodeCamp(YouTube>Web3)

### LINKED ISSUE #438: [BUG] Consistent and Proper Spelling of Org and Language
### Is there any existing issue for this?

- [X] I have searched the existing issues

### Description

Noticed the spelling mistakes here. 

### Screenshots

- CSS spelling [YouTube Section]
Screenshot 2023-04-14 at 14 02 15

- FreeCodeCamp [YouTube Section > Web3]
Screenshot 2023-04-14 at 14 00 55

### Any additional information?

_No response_

### What browser are you seeing the problem on?

Chrome

### PR DESCRIPTION
**Pull request by TreciaKS**
## Fixes Issue
Closes #438 

## Changes proposed
- Capitalised CSS
- Capitalised 'f' in FreeCodeCamp

## Screenshots

-  Location: LinksHub/database/data.ts
Screenshot 2023-04-14 at 14 57 29

- Location: LinksHub/database/youtube/web3-metaverse.json
Screenshot 2023-04-14 at 14 59 32

### CODE REVIEW

File: database/youtube/web3-metaverse.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
         "language": "english"
     },
     {
-        "name": "freeCodeCamp.org",
-        "description": "freeCodeCamp is a superb weebsite that aims at providing quality ontent for free across all domains and it provides quality content for this domain too.",
+        "name": "FreeCodeCamp.org",
+        "description": "FreeCodeCamp is a superb weebsite that aims at providing quality ontent for free across all domains and it provides quality content for this domain too.",
```
Comment (rupali-codes) [inline database/youtube/web3-metaverse.json:20]:
```suggestion
        "description": "FreeCodeCamp is a superb weebsite that aims at providing quality ontent for free across all domains and it provides quality content for this domain too.",
```
```suggestion
        "description": "FreeCodeCamp is a superb website that aims at providing quality content for free across all domains and it provides quality content for this domain too.",
```

**Review by rupali-codes — COMMENTED** [review]
hey @TreciaKS just fixed some typos, then we are good to go!

**Comment by TreciaKS** [comment]
@rupali-codes Done ✅

**Review by ujjawaltyagii — CHANGES_REQUESTED** [review]
Hi @TreciaKS , you are doing great but there is still a typo in ``Web3 & Metaverse`` category in ``FreeCodeCamp.org`` you can update your PR after doing the necessary changes.
(content instead of ontent)
image

**Comment by TreciaKS** [comment]
@ujjawaltyagii Thanks! I've fixed necessary spelling and grammar.

File: database/youtube/web3-metaverse.json [hunk 2]
```diff
@@ -16,8 +16,8 @@
         "language": "english"
     },
     {
-        "name": "freeCodeCamp.org",
-        "description": "freeCodeCamp is a superb weebsite that aims at providing quality ontent for free across all domains and it provides quality content for this domain too.",
+        "name": "FreeCodeCamp.org",
+        "description": "FreeCodeCamp is a superb website that aims at providing quality content for free across all domains and it provides quality content for this domain too.",
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:20]:
```suggestion
        "name": "FreeCodeCamp.org",
        "description": "FreeCodeCamp is a superb website that aims to provide free high-quality tech content across all domains.",
```

File: database/youtube/web3-metaverse.json [hunk 3]
```diff
@@ -26,10 +26,19 @@
     {
         
         "name": "Eat the Blocks",
-        "description": "Eat the Block is yet another a youtube channel with some in depth content on various topics on Web3 and Blockchain. It is one of those channels that you should definitely give a try if you want to step into this emerging field of Web3.",
+        "description": "Eat the Block is yet another YouTube channel with some in depth content on various topics on Web3 and Blockchain. It is one of those channels that you should definitely give a try if you want to step into this emerging field of Web3.",
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:29]:
```suggestion
        "description": "Eat the Block is yet another YouTube channel with some in depth content on various topics on Web3 and Blockchain. It is one of those channels that you should definitely give a try if you want to step into this emerging field of tech.",
```

File: database/youtube/web3-metaverse.json [hunk 4]
```diff
@@ -26,10 +26,19 @@
...
         "url": "https://www.youtube.com/@EatTheBlocks",
         "category": "youtube",
         "subcategory": "web3-metaverse",
         "language": "english"
+      },
+     {
+
+        "name": "Francesco Cuilla",
+        "description": "Francesco's YouTube channel discusses topics about being in DevRel, Docker, Web3 and Bloackchain.",
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:38]:
```suggestion
        "description": "Francesco's YouTube channel discusses a wide range of tech topics such as DevRel, Docker, Web3, and Blockchain.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @TreciaKS! 😄 I made a few suggestions on how to make your descriptions clearer. In the future, I highly recommend you run them through [Grammarly](https://www.grammarly.com/browser)before doing a PR. Other than that great work! 😄

**Comment by TreciaKS** [comment]
Hey Christine! Thank you! Your suggestions are pretty awesome. Committed the changes ✅

**Review by CBID2 — APPROVED** [review]
Congratulation, You just made your first PR!🥳 LGTM! 😁

**Review by rupali-codes — APPROVED** [review]
Looks perfect to me, thanks @TreciaKS

**Pull request merged**